src/templates.ts

```diff
@@ -41,10 +41,10 @@ export const htmlTemplate = (name: string, imports: string[], js: string, css: s
     </style>
 </script>
 ${imports.map((i) => `<script type="text/html" import-script="${i}" import-async="false"></script>\n`)}
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/a22x/${name}/template.js" import-async="false"></script>
+<script type="text/html" import-script="${packagesDir}/${name}/template.js" import-async="false"></script>
```

Comment:
Just a minor thing, the root of the import has to be `/Pages`, but with this configuration it would be `/PackageSources`.
```suggestion
<script type="text/html" import-script="/Pages/VCockpit/Instruments/${process.env.PACKAGE_NAME}/${name}/template.js" import-async="false"></script>
```